react-app/src/components/schemaFormUtils.js

```diff
@@ -67,8 +70,130 @@ const uiSchema = {
   },
 };
 
+function Extractor(props) {
+  return (
+    <Accordion.Item eventKey={props.eventKey}>
+      <Accordion.Header>{props.formData.type}</Accordion.Header>
+      <Accordion.Body>
+        <p>This is a placeholder for an extractor. An input form will be added here.</p>
+      </Accordion.Body>
+    </Accordion.Item>
+  );
+}
+
+function ExtractorArray(props) {
+  const [extractors, setExtractors] = useState([]);
+  const [extractorsJSX, setExtractorsJSX] = useState([]);
+  const [types, setTypes] = useState([
+    'BaseFHIRExtractor',
+    'CSVAdverseEventExtractor',
+    'CSVCancerDiseaseStatusExtractor',
+    'CSVCancerRelatedMedicationExtractor',
+    'CSVClinicalTrialInformationExtractor',
+    'CSVConditionExtractor',
+    'CSVObservationExtractor',
+    'CSVPatientExtractor',
+    'CSVProcedureExtractor',
+    'CSVStagingExtractor',
+    'CSVTreatmentPlanChangeExtractor',
+    'Extractor',
```

Comment:
Since the Extraction UI only works with CSV Extraction, you could probably cut this list down to just any CSV[blank]Extractors. At the very least, you can definitely remove "Extractor" because that's just a base class that all other extractors extend, it doesn't actually do anything.

Reply:
I have removed the extra extractors.